data/management/commands/import_contributors_data.py

```diff
@@ -37,13 +39,19 @@ def handle(self, *args, **options):
         """
         Makes a GET request to the  API.
         """
+        logger = logging.getLogger(__name__)
         headers = {'Content-Type': 'application/json'}
-        response = requests.get(
-            url=IMPORT_URL,
-            headers=headers,
-        )
 
-        response.raise_for_status()
+        try:
+            response = requests.get(
+                url=IMPORT_URL,
+                headers=headers,
+            )
+            response.raise_for_status()
+        except Exception as e:
+            logger.error(e)
+            return
```

Comment:
`return`? seems very unusual.

Reply:
What should a django `Command.handle` do when there is an error ?